tasklist/pom.xml

```diff
@@ -911,415 +125,58 @@
+            </excludes>
+          </configuration>
         </plugin>
-
       </plugins>
-
     </pluginManagement>
-
-    <plugins>
-      <plugin>
-        <groupId>org.sonatype.plugins</groupId>
-        <artifactId>nexus-staging-maven-plugin</artifactId>
-      </plugin>
-
-      <plugin>
-        <groupId>org.apache.maven.plugins</groupId>
-        <artifactId>maven-enforcer-plugin</artifactId>
-      </plugin>
-
-      <plugin>
-        <groupId>org.apache.maven.plugins</groupId>
-        <artifactId>maven-dependency-plugin</artifactId>
-      </plugin>
-
-      <plugin>
-        <groupId>org.jacoco</groupId>
-        <artifactId>jacoco-maven-plugin</artifactId>
-        <configuration>
-          <excludes>
-            <!--exclusions for `tasklist-common` module-->
-            <exclude>**/camunda/tasklist/entities/**/*</exclude>
-            <exclude>**/camunda/tasklist/exceptions/**/*</exclude>
-            <exclude>**/camunda/tasklist/property/**/*</exclude>
-            <!--exclusions for `tasklist-webapp` module-->
-            <exclude>**/camunda/tasklist/webapp/api/rest/v1/entities/**/*</exclude>
-            <exclude>**/camunda/tasklist/webapp/management/dto/**/*</exclude>
-            <exclude>**/camunda/tasklist/webapp/graphql/entity/**/*</exclude>
-            <exclude>**/camunda/tasklist/webapp/rest/exception/**/*</exclude>
-            <exclude>**/camunda/tasklist/webapp/config/**/*</exclude>
-            <exclude>**/camunda/tasklist/webapp/es/dao/response/**/*</exclude>
-            <exclude>**/camunda/tasklist/webapp/**/*ErrorController.*</exclude>
-            <exclude>**/camunda/tasklist/webapp/**/ClusterMetadata.*</exclude>
-          </excludes>
-        </configuration>
-        <executions>
-          <execution>
-            <id>prepare-agent</id>
-            <goals>
-              <goal>prepare-agent</goal>
-            </goals>
-          </execution>
-        </executions>
-      </plugin>
-    </plugins>
   </build>
 
   <profiles>
-
-    <!-- profile to auto format -->
     <profile>
-      <id>autoFormat</id>
-      <activation>
-        <activeByDefault>true</activeByDefault>
-      </activation>
+      <!--
+        We have to skip spotbugs analysis until we fix several findings
+        See https://github.com/camunda/zeebe/issues/17598
+      -->
+      <id>spotbugs</id>
       <build>
         <plugins>
           <plugin>
-            <groupId>com.diffplug.spotless</groupId>
-            <artifactId>spotless-maven-plugin</artifactId>
-            <executions>
-              <execution>
-                <id>spotless-format</id>
-                <goals>
-                  <goal>apply</goal>
-                </goals>
-                <phase>process-sources</phase>
-              </execution>
-            </executions>
+            <groupId>com.github.spotbugs</groupId>
+            <artifactId>spotbugs-maven-plugin</artifactId>
+            <configuration>
+              <skip>true</skip>
```

Comment:
❓ Do we have an issue to work on this?

Reply:
yes https://github.com/camunda/zeebe/issues/17598